docs/resources/aws_batch_job_definitions.md

```diff
@@ -0,0 +1,79 @@
+---
+title: About the aws_batch_job_definitions Resource
+platform: aws
+---
+
+# aws\_batch\_job\_definitions
+
+Use the `aws_batch_job_definitions` InSpec audit resource to test properties of a plural Batch Job Definition.
+
+The AWS::Batch::JobDefinition resource specifies the parameters for an AWS Batch job definition.
+
+## Syntax
+
+Ensure that a job_definition exists.
+    describe aws_batch_job_definitions do
+      it { should exist }
+    end
+
+## Parameters
+
+For additional information, see the [AWS documentation on Batch Job Definition](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-athena-workgroup.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| job_definition_names | The name of the job definition. |
+| job_definition_arns | The arn of the job definition. |
+| revisions | The revision of the job definition. |
+| statuses | The status of the job definition. |
+| types | The type of the job definition. |
+| parameters | The parameter of the job definition. |
+| tags | The tags of the job definition. |
+| propagate_tags | The propagate_tags of the job definition. |
+| platform_capabilities | The platform_capabilities of the job definition. |
+
+## Examples
+
+### Ensure a job_definition name is available.
+    describe aws_batch_job_definitions do
+      its('job_definition_names') { should include 'test1' }
+    end
+
+### Ensure that the state is `ENABLED` or `DISABLED`.
+    describe aws_batch_job_definitions do
+        its('statuses') { should include 'ACTIVE' }
+    end
```

Comment:
Is this right? Should the heading be: Ensure that the status is `ACTIVE`.

Reply:
Yes.